huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_instance_test.go

```diff
@@ -31,6 +31,8 @@ func TestAccInstance_basic(t *testing.T) {
 		rName        = acceptance.RandomAccResourceName()
 		updateName   = acceptance.RandomAccResourceName()
 	)
+	srcEPS := acceptance.HW_ENTERPRISE_PROJECT_ID_TEST
+	destEPS := acceptance.HW_ENTERPRISE_MIGRATE_PROJECT_ID_TEST
```

Comment:
```suggestion
		rName        = acceptance.RandomAccResourceName()
		updateName   = acceptance.RandomAccResourceName()
		epsId        = acceptance.HW_ENTERPRISE_PROJECT_ID_TEST
	)
```
invalid format for parameter definition, and there is unnecessary to use two eps ID variables, default enterprise project (0) we can use.

Reply:
I understand, it has been corrected

Reply:
I have switched to using environment variables.